### ISSUE TITLE
Illegal address error when trying to send e-mail

### ISSUE THREAD

**Issue by levyj** [socrata/datasync#51]
Command Line from Linux:

Error sending email to: dataportal@cityofchicago.orgIllegal addressjava.lang.IllegalArgument

Relevant line from config.json:

"adminEmail": "dataportal@cityofchicago.org",

**Comment by jeffcoh** [comment]
I found this issue via a Google search.

I am running into the same issue with DataSync. I just updated to 1.8.0 and still getting the "illegal address" when clicking "Test SMTP Settings". If I skip the test and DataSync throws an error, I can see the same "illegal address" error in the console output.

I've tried a variety of different email addresses, all produce the same error.

Running WireShark on the machine at the same time as clicking "Test SMTP Settings" does not show it producing any network activity on port 25.

In the Preferences box I have the admin email, outgoing mail server, and smtp port specified. None of the other mail related settings are required in my environment.
